chapters/introduction.typ

```diff
@@ -19,17 +19,16 @@ To support the C++ community with its refactoring tasks, new refactoring feature
 
 == Initial Situation
 
-// COR strictly speaking it is not only about types
 The programming language C++ is constantly developed further by the standard committee @cpp_standard_committee. 
-With C++20 type constraints were introduced which allow specification of requirements for template arguments.
+With C++20 template parameter constraints were introduced which allow specification of requirements for template arguments.
 Concepts are a way of abstracting these constraints. @constraints_and_concepts
 
 Refactoring is a common technique to resolve code smells and improve the internal structure of code without changing its external behavior. @refactoring
 Automated tests often ensure that the correct functionality is retained.
 
 Older versions of integrated development environments (IDE) were implementing support for code analysis and tools like symbol lookup and refactorings themselves.
-// COR And also that every tool had to spend the effort to implement the whole support.
-This led to the problem that new languages only slowly gained adoption, one editor at a time.
+This led to the problem that new languages only slowly gained adoption, one editor at a time and also that every tool had to spend the effort to implement the whole support.
+Another downside was that every tool had to spend the effort to implement the whole support on their own.
```

Comment:
```suggestion
This led to the problem that new languages only slowly gained adoption, one editor at a time, each of them having to spend the effort to implement support for it.
```

Reply:
wenn de satz wiedre usechunt isch de comment ned resolved:
// COR And also that every tool had to spend the effort to implement the whole support.

Reply:
"each of them having to spend the effort to implement support for it." langet doch ned?